pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR/files/Microsoft.NETCore.UniversalWindowsPlatform.CoreCLR.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildDependsOn>
+      CheckCoreRuntimeCompatibility;
+      $(BuildDependsOn);
+    </BuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CheckCoreRuntimeCompatibility">
+    <Error
```

Comment:
I have already tested this with the latest RS2 Win 10 SDK (10.0.15063.0) and it fails correctly. I wanted to check that we indeed wanted this to fail for current scenarios and therefore are only enabling it for future scenarios.